### ISSUE TITLE
5.2.3 breaks CSS Custom Properties

### ISSUE THREAD

**Issue by romainmenke** [clean-css/clean-css#1196]
### Environment

* clean-css version - `npm ls clean-css`: v5.2.3
* node.js version - `node -v`: 16
* operating system: mac os

### Configuration options

```js
new CleanCSS( {
	sourceMap: true,
} ).minify( result.css, JSON.parse( result.map ), ( error, output ) => {
	if ( error ) {
		reject( error );

		return;
	}

	resolve( output );
} );
```

### Input CSS

```css
:root {
	/* Site container */
	--a: 20px;

	/* Z indices for modals and dialogs */
	--foo: 5050;
}

.bar {
	z-index: var(--foo);
}
```

### Actual output CSS

```css
:root{--a:20px}.bar{z-index:5050;z-index:var(--foo)}
```

Notice that `--foo:5050` is missing in the actual output

### Expected output CSS

```css
:root{--a:20px;--foo:5050}.bar{z-index:5050;z-index:var(--foo)}
```

This is taken from clean-css 5.2.2

**Comment by geierm** [comment]
I have the same problem and could nail it down the comment lines. Interestingly custom properties with comments before will disappear in the minified code. But not all lines ;-).

```css
:host {
      /* this will stay */
      --this_will_stay: 1;
  
      /* this will disappear */
      --this_will_disappear: 1;
  
      /* this will also disappear */
      --this_will_also_disappear: 1;
  
      --this_will_stay_aswell: 1;
  
      --this_will_stay_too: 1;
  
      width: 1px;
      height: 1px;
  }
```
With version 5.2.3 this compiles to:

```css
:host{--this_will_stay:1;--this_will_stay_aswell:1;--this_will_stay_too:1;width:1px;height:1px}
```

**Comment by jakubpawlowicz** [comment]
It's fixed in 5.2.4. Sorry for trouble.

**Comment by romainmenke** [comment]
Thank you for fixing this so quickly!
